### ISSUE TITLE
"Account creation on ACMEv1 is disabled"

### ISSUE THREAD

**Issue by tomgidden** [JustinAiken/jekyll-gitlab-letsencrypt#28]
My regular gitlab-ci cert renewal pipeline's failing now, thanks to the deprecation of ACMEv1:

```
$ bundle exec jekyll letsencrypt
Configuration file: /builds/xxxxxx/xxxxxx/_config.yml
Registering xxxxxx@xxxxxx.xxx to https://acme-v01.api.letsencrypt.org/... 
jekyll 3.8.6 | Error:  Account creation on ACMEv1 is disabled. Please upgrade your ACME client to a version that supports ACMEv2 / RFC 8555. See https://community.letsencrypt.org/t/end-of-life-plan-for-acmev1/88430 for details.
ERROR: Job failed: exit code 1
```

I did have a go switching to `acme-client 2.0.5`, but with absolutely _zero_ knowledge of Ruby, Jekyll, gems, etc. I wasn't able to get very far with it, unfortunately.

**Comment by tomgidden** [comment]
Incidentally, I just realised GitLab Pages now offers [automatic Let's Encrypt certificates](https://gitlab.com/help/user/project/pages/custom_domains_ssl_tls_certification/lets_encrypt_integration.md), so I'm not sure if this project is still required.  Anyway, it might be worth just pointing this new capability out instead of fixing this issue.

**Comment by JustinAiken** [comment]
Uh oh, that means this project is totally hosed for now.  I'll give the new automatic thing a try - if it works for a standard jekyll blog I'll leave a note in the README...

**Comment by tothlp** [comment]
Hey there! I wrote an email yesterday, but I'm really pleased to find this link. Thanks a lot! I'm just giving it a try. :)

**Comment by tothlp** [comment]
Gitlab's automatic magic works! :)

**Comment by JustinAiken** [comment]
Yep, Gitlab's automation worked like a charm for me too.  This whole project is probably unnecessary now 🎊 

I added a note about that at the top of the project page: https://github.com/JustinAiken/jekyll-gitlab-letsencrypt#deprecated

**Comment by tomgidden** [comment]
Agreed, but thank you @JustinAiken for getting it to work before the magic tickbox was implemented! 😁 


(Incidentally, I haven't closed the issue, in case you want to leave it open and pinned for prominence, but feel free to do so)

**Comment by JustinAiken** [comment]
Yeah, I'll keep it open for the big pin - thanks for reporting though, I wouldn't have noticed until I went to renew my cert sometime in late Dec!
